usb/libloader.py

```diff
@@ -66,7 +67,7 @@ class LibraryMissingSymbolsException(LibraryException):
     pass
 
 
-def locate_library (candidates, find_library=ctypes.util.find_library):
```

Comment:
This looks redundant with

```py
    if find_library is None:
        find_library = ctypes.util.find_library
```

but isn't _in intention_.  At least I think it isn't (I'm not the original author).

Technically, the conditional alone would be enough, of course, but having an obvious default is valuable...  Probably more than supporting the case where the caller is unable to decide whether to include the parameter or not, thus relying on `None` to signal "the default, please".

I'm leaning towards keeping the minor redundancy in favor or expressiveness.  Do you mind reverting this part?

Reply:
No problem